internal/commands/result.go

```diff
@@ -199,6 +208,7 @@ func resultShowSubCommand(
 		printer.FormatPDF,
 		printer.FormatSummaryMarkdown,
 		printer.FormatGL,
+		// printer.FormatSonar ??
```

Comment:
what is this comment?

Reply:
we support also this format, I don't know why it is not on the list already